runtime/src/dice.rs

```diff
@@ -233,17 +243,23 @@ pub fn copy_rt_alias_cert(
     let tbs = persistent_data
         .rtalias_tbs
         .get(..persistent_data.fht.rtalias_tbs_size.into());
-    cert_from_tbs_and_sig(tbs, &persistent_data.fht.rt_dice_sign, cert)
+    cert_from_tbs_and_sig(
+        tbs,
+        &persistent_data.fht.rt_dice_sign,
+        cert,
+        CaliptraError::RUNTIME_GET_RT_ALIAS_CERT_FAILED,
+    )
 }
 
 /// Create a certificate from a tbs and a signature and write the output to `cert`
 fn cert_from_tbs_and_sig(
     tbs: Option<&[u8]>,
     sig: &Ecc384Signature,
     cert: &mut [u8],
+    err: CaliptraError,
```

Comment:
nit: instead of changing the function signature, could use `map_err` at the callsites. Might be a little more clear what is happening.

Reply:
Since this is the only comment, I'm happy to just merge this anyway. Up to you.

Reply:
Since I followed this pattern in two places, decided to just make the change. Reads a lot simpler too.